Addresses typos, word choice, and sentence structure

paladp opened the pull request.

paladp committed:
Addresses typos, word choice, and sentence structure

paladp committed:
Fixes trailing spaces

sutterj commented on assessment.md:
```diff
@@ -27,23 +27,23 @@ when creating python programs.
-    and troubleshooting to other students regarding their own development
+  * E = Additional development tools configured and ability to provide assistance
+    and troubleshooting to other individuals regarding their own development
     environment
```
I would say either:
...additional tools configured and is able to...
or
...additional tools configured and has the ability to...

ilikerustoo replied:
We should remove "being able to" since this assessment is about what has been done rather than what can be done. We should not be based on being able to help others as this would be difficult to prove and unnecessary.

ilikerustoo commented on assessment.md:
```diff
@@ -27,23 +27,23 @@ when creating python programs.
   * E = Able to run programs with `pyenv` and `pipenv` using additional commands
     and flags to streamline workflow, while being able to provide assistance and
-    troubleshooting to other students using `pyenv` and `pipenv`
+    troubleshooting to other individuals using `pyenv` and `pipenv`
```
We should remove "being able to" since this assessment is about what has been done rather than what can be done. We should not be based on being able to help others as this would be difficult to prove and unnecessary.

ilikerustoo commented on assessment.md:
```diff
@@ -53,25 +53,25 @@ when creating python programs.
-  * I = Incorrectly attempted docstrings unidentifiable to compiler
-  * A = Added docstrings, but it does not fully describe the functions'/methods'
+  * I = Incorrectly attempted docstrings unidentifiable to the compiler
+  * A = Added docstrings, but they do not fully describe the functions'/methods'
```
Find better word than "attempted"

ilikerustoo reviewed (dismissed):
Some wording changes are needed

huangs1 committed:
Resolve some issues brought up by reviewers

Change the part that @sutterj suggested we should change and remove all the "being able to" parts. Also, found a better word than "attempted" as suggested by @ilikerustoo.

huangs1 reviewed (approved):
I think that the changes look good. Thanks for the hard work!

Michionlion committed:
Merge branch 'master' into patch-5
